.github/workflows/test-windows.yml

```diff
@@ -9,7 +9,7 @@ on:
       - "**.svg"
       - "adr/**"
       - "docs/**"
-      - "CODEOWNERS"      
```

Comment:
how do you even make these spaces @jeff-mccoy ?

Reply:
😂 no idea...that was an autoformat...

Reply:
wait... no that was from _YOUR_ pr.... lol